examples/python/transfer_l2_usdc.py

```diff
@@ -0,0 +1,137 @@
+import asyncio
+import logging
+import os
+from typing import Dict
```

Comment:
from python 3.9, you can use `dict` as type annotation instead of `Dict` that you import here